### PR TITLE
Add features as tags

### LINKED ISSUE #273: Add features automatically as tags

### PR DESCRIPTION
**Pull request by ilopmar**
Fixes #273

### CODE REVIEW

File: buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy [hunk 1]
```diff
@@ -269,4 +270,17 @@ class IndexGenerator {
         return JsonOutput.toJson(result)
     }
 
+    @CompileDynamic
+    private static List<String> generateTags(GuideMetadata guide) {
+        [
+            guide.tags +
+            guide.languages.collect { it.toString() } +
```
Comment (sdelamo) [inline buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy]: Please replace this with `guide.languages*.toString()`

File: buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy [hunk 2]
```diff
@@ -269,4 +270,17 @@ class IndexGenerator {
...
+            guide.buildTools.collect { it.toString() } +
```
Comment (sdelamo) [inline buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy]: Please, replace this with `guide.buildTools*.toString()`

**Review by sdelamo — APPROVED** [review]

File: buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy (hunk 1 above)
Comment (ilopmar) [reply buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy]: done

File: buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy (hunk 2 above)
Comment (ilopmar) [reply buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy]: done

**Pull request merged**